gapic/src/main/com/google/gapid/perfetto/models/CounterInfo.java

```diff
@@ -27,27 +27,44 @@
 import com.google.common.util.concurrent.ListenableFuture;
 import com.google.gapid.models.Perfetto;
 import com.google.gapid.perfetto.Unit;
+import com.google.gapid.perfetto.models.QueryEngine.Row;
```

Comment:
there are now mixed usages of 'Row' and 'QueryEngine.Row' in this file, could be better to keep using QueryEngine.Row as 'Row' is quite ambiguous.

Reply:
SGTM. Modified.